source/index.md

```diff
@@ -18,9 +18,9 @@ search: true
 ---
 
 # Introduction
+# Introduction
 
-Welcome to the Planning Center Online API! You can use our API to access data in some of our apps
-(with other apps coming online soon) through a single unified API.
+The Planning Center Online API can be used to interact with all of your organization's data inside Planning Center. In this first iteration of the API the data inside PCO People is available; we are now working towards adding endpoints for each of our other apps (including an all new API for Services).
```

Comment:
Avoid promises. Services is a reasonable promise, but Registrations probably won't have an API anytime soon :)